src/components/Interval.js

```diff
@@ -15,7 +15,7 @@ class Interval extends Component {
   }
 
   _setIntervalIfNecessary = delay => {
-    if (Number.isFinite(delay)) {
+    if (Number.isFinite(delay) && !this.intervalId) {
```

Comment:
I think removing the check `&& !this.intervalId` and adding `this._clearIntervalIfNecessary()` right after solve the problem. Can you do that so we can merge PR?

Reply:
done~